### ISSUE TITLE
How do I change the state color in the indicators ?

### ISSUE THREAD

**Issue by DaN66O** [artem-sedykh/mini-climate-card#120]
Can I use styles or mapper to change the color of the value in the  indicators or hide state ?

![Screenshot_18](https://user-images.githubusercontent.com/113509164/236615570-93c00c7a-d6f8-43a8-acca-db13e4750274.png)

Actually I wanted to show only the icon without showing the value.
I thought it could be done using
` .state__value { color: 'transparent !important' }`
or
` .state__value {display: none;}`

I have changed the icon and its color, but I do not know how to make the value invisible.
```
    indicators:
      t_sensor:
        icon:
          template: >
            (value) => (value == 'Ext' ? 'mdi:alpha-e-circle-outline' : 'mdi:alpha-i-circle-outline')
          style: > 
            (value) => (value == 'Ext' ? { color:'#4bc241'} : {color:'#b74a9b'} )
```

**Comment by regevbr** [comment]
This is currently not supported. A PR is more than welcome!

About
```
 .state__value { color: 'transparent !important' }
```
This should work but I'm not sure how you can apply custom CSS to an isolated component

**Comment by DaN66O** [comment]
`.state__value { color: 'transparent !important' }` and `.state__value {display: none;}`
It doesn't work
It would be ideal if it was possible to hide the value leaving only the icon in the indicators.
Can we hope that this will be done in the next versions ?

**Comment by regevbr** [comment]
If you create a PR, then yes I will release it as soon as possible after a code review

**Comment by DaN66O** [comment]
I can't create a PR.
Tell me how ?

**Comment by regevbr** [comment]
https://docs.github.com/en/get-started/quickstart/contributing-to-projects

**Comment by DaN66O** [comment]
I read the link above
I realized that I myself have to make changes to the project and post them.
But I do not know how to make these changes...

**Comment by regevbr** [comment]
released in 2.6.0

**Comment by regevbr** [comment]
you can use `indicator:name:value:style`, same as you do for icon
